api/src/server/documents/sheets/cells/controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Request } from "hapi";
+
+import * as Cell from "../../../../data/Cell/service";
```

Comment:
Do we not have a better way to get imports from nested directories? This feels fragile.

Reply:
I agree, though my first attempt generated missing module errors. I'll give it another shot.

Reply:
Should be OK for now and can be addressed later for the project as a whole.